### ISSUE TITLE
Error compiling on rust 1.0

### ISSUE THREAD

**Issue by psychoticmeow** [nickel-org/nickel.rs#219]
I followed the instructions in the readme to add nickel to my project, and this happened:

```
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:23:1: 27:2 error: the trait `core::marker::Reflect` is not implemented for the type `T` [E0277]
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:23 pub struct Response<'a, T=Fresh> {
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:24     ///the original `hyper::server::Response`
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:25     pub origin: HyperResponse<'a, T>,
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:26     templates: &'a TemplateCache
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:27 }
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:23:1: 27:2 note: `T` does not implement `Any`; ensure all type parameters are bounded by `Any`
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:23 pub struct Response<'a, T=Fresh> {
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:24     ///the original `hyper::server::Response`
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:25     pub origin: HyperResponse<'a, T>,
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:26     templates: &'a TemplateCache
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:27 }
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:23:1: 27:2 error: the parameter type `T` may not live long enough [E0310]
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:23 pub struct Response<'a, T=Fresh> {
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:24     ///the original `hyper::server::Response`
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:25     pub origin: HyperResponse<'a, T>,
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:26     templates: &'a TemplateCache
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:27 }
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:23:1: 27:2 help: consider adding an explicit lifetime bound `T: 'static`...
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:23:1: 27:2 note: ...so that the type `T` will meet its required lifetime bounds
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:23 pub struct Response<'a, T=Fresh> {
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:24     ///the original `hyper::server::Response`
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:25     pub origin: HyperResponse<'a, T>,
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:26     templates: &'a TemplateCache
/Users/rowan/.cargo/registry/src/github.com-1ecc6299db9ec823/nickel-0.3.0/src/response.rs:27 }
error: aborting due to 2 previous errors
Could not compile `nickel`.
```

**Comment by Ryman** [comment]
@cburgdorf Can you add me as owner on 'crates.io'? [The commit that fixes this](https://github.com/nickel-org/nickel.rs/commit/1f30e68a941266a9bcae2b71e0923b8ab500f625) has been on master for 10days :(

**Comment by cburgdorf** [comment]
Sorry about that. I just added you as another owner of crates.io. Should I just publish the latest of master?

**Comment by cburgdorf** [comment]
OK, should be fixed with version 0.4.0 https://crates.io/crates/nickel

**Comment by Ryman** [comment]
@cburgdorf Thanks :+1:
